Explorer/Assets/DCL/SDKComponents/AvatarShape/Systems/AvatarShapeHandlerSystem.cs

```diff
@@ -70,6 +70,7 @@ private void HandleComponentRemoval(in Entity entity, ref SDKAvatarShapeComponen
         private void HandleEntityDestruction(in Entity entity, ref SDKAvatarShapeComponent sdkAvatarShapeComponent)
         {
             World.Remove<SDKAvatarShapeComponent>(entity);
+            World.Remove<PBAvatarShape>(entity);
```

Comment:
Same here: double deletion should be replaced with a single deletion with 2 arguments